projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/container/multi-step-checkout.component.spec.ts

```diff
@@ -126,7 +127,12 @@ describe('MultiStepCheckoutComponent', () => {
         MockReviewSubmitComponent,
         MockShippingAddressComponent
       ],
-      providers: [CheckoutService, CartService, CartDataService]
+      providers: [
+        CheckoutService,
+        CartService,
+        CartDataService,
+        GlobalMessageService
```

Comment:
I think we should mock GlobalMessageService in every unit test (except GlobalMessageService's own unit tests, of course).  It aligns with the principle of unit testing, and maybe more importantly, the store is injected in GlobalMessageService and if we don't use a mock for GlobalMessageService, we will still need to import the store to fulfil the dependency injection defined in GlobalMessageService's constructor.

This comment applies to all the unit tests in this PR.

Reply:
We will refactor these tests when we work on the facades of those modules. In this case, when we add facades in checkout module, we will refactor this test.